tests/errmsgs/t10735.nim

```diff
@@ -2,25 +2,26 @@ discard """
   cmd: "nim check $file"
   errormsg: "selector must be of an ordinal type, float or string"
   nimout: '''
-t10735.nim(38, 5) Error: 'let' symbol requires an initialization
-t10735.nim(39, 10) Error: undeclared identifier: 'pos'
-t10735.nim(39, 9) Error: type mismatch: got <cstring, >
+t10735.nim(39, 5) Error: 'let' symbol requires an initialization
```

Comment:
2 remaining cases:
```
proc `[]`*[T, U](n: NimNode, x: HSlice[T, U]): seq[NimNode] =
iterator items*[T](s: HSlice[T, T]): T =
```
(the latter because `for ai in 0.3 .. 3.5: echo ai` would give an error when instantiated)